src/soar_sdk/app.py

```diff
@@ -19,20 +19,54 @@
 from soar_sdk.logging import getLogger
 from soar_sdk.exceptions import ActionFailure, AssetMisconfiguration
 import traceback
+import uuid
+
+
+def is_valid_uuid(value: str) -> bool:
+    """Validates if a string is a valid UUID"""
+    try:
+        return str(uuid.UUID(value)).lower() == value.lower()
+    except ValueError:
+        return False
 
 
 class App:
     def __init__(
         self,
         *,
         name: str,
+        app_type: str,
+        logo: str,
+        logo_dark: str,
+        product_vendor: str,
+        product_name: str,
+        publisher: str,
+        appid: str,
+        python_version: list[str] = ["3.9", "3.13"],  # noqa: B006
```

Comment:
You can remove the `noqa` like this:

```suggestion
        python_version: Optional[list[str]] = None
```

and then in the function, do

```python
if python_version is None:
    python_version = ["3.9", "3.13"]

Reply:
did `python_version or ["3.9", "3.13"]` in self.app_meta_info instead 👍